core/domain/exp_services.py

```diff
@@ -421,6 +563,19 @@ def export_to_zip_file(exploration_id, version=None):
 
 
 def convert_state_dict_to_yaml(state_dict, width):
+    """Converts the given state dict to yaml format
+
+    Args:
+        state_dict: dict. A dict representing state of exploration.
```

Comment:
Nit: In order to avoid confusion between the various meanings of "state", I suggest "representing a state in an exploration".